tests/test_tool.py

```
@@ -341,8 +341,38 @@ def test_get_exe_empty(running_node):
+    assert "Missing tool can be installed via: \"sc-install builtin\"" in caplog.text
+
+
+def test_get_exe_not_found_no_suggestion(running_node, monkeypatch, caplog):
+    monkeypatch.setattr(running_node.project, "_Project__logger", logging.getLogger())
+    running_node.project.logger.setLevel("INFO")
+
+    assert running_node.project.set('tool', 'builtin', 'task', 'nop', 'exe', 'testexe')
+    with running_node.task.runtime(running_node) as runtool:
+        with patch("json.load") as load:
+            load.return_value = {"nothis": {}}
+            with pytest.raises(TaskExecutableNotFound, match="^testexe could not be found$"):
```

Ruff (0.13.3)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)